package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@wikimedia/mw-node-qunit",
-	"version": "6.2.1",
+	"version": "6.3.0",
```

Comment:
You need to update package-lock.json too (run `npm i`)

Reply:
updated, silly me